src/user/user.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Controller, Get, Post, Body, Param, Delete, UsePipes, ValidationPipe, Put, HttpCode } from '@nestjs/common'
+import { UserService } from './user.service'
+import { CreateUserResponse, GetUserDto, UpdatePersonalInfoDto, UpdateRelationshipDto } from './dto/user.dto'
+import { CreateUserDto } from './dto/user.dto'
+import { ApiBadRequestResponse, ApiOkResponse, refs } from '@nestjs/swagger'
+
+@Controller('user')
+export class UserController {
+  constructor(private readonly userService: UserService) {}
+  @Get(':id')
```

Comment:
I am not sure where your uid came from. Just want to remind that if it came from jwt, the uid should be got by @Req() req. (But ask ping again about how to get uid from jwt, it is up to what type of jwt he used)